barretenberg/cpp/src/barretenberg/vm2/tracegen_helper.cpp

```diff
@@ -48,6 +51,11 @@ auto build_precomputed_columns_jobs(TraceContainer& trace)
             AVM_TRACK_TIME("tracegen/precomputed/range_16", precomputed_builder.process_sel_range_16(trace));
             AVM_TRACK_TIME("tracegen/precomputed/power_of_2", precomputed_builder.process_power_of_2(trace));
         },
+        [&]() {
+            PrecomputedTraceBuilder precomputed_builder;
+            AVM_TRACK_TIME("tracegen/precomputed/round_constants",
+                           precomputed_builder.process_sha256_round_constants(trace));
```

Comment:
This is 64 rows, right? Probably can go inside one of the other jobs.

Reply:
what's the rational for the jobs splitting? Should jobs that will run over similar rows be grouped together? (Just thinking ahead for stuff like keccak round constants (which is also "small" - 24 rows)

Reply:
Whatever makes sense really, but the rule of thumb is that different jobs should work on different _columns_ (rows don't really matter).